tests/unit_tests/test_wave_06.py

```diff
@@ -75,17 +75,23 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
-
-@pytest.mark.skip
+    
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
Great start! It would be a good idea to also check to see if the other items are still in the inventories like we expect.